### DISCUSSION TITLE
Official Docker image

### DISCUSSION THREAD

**Discussion by mnaser** [ansible/ansible-lint#2374] [Ideas] [answered]
👋 

One of the issues that I'm currently dealing with is a lack of an up-to-date `ansible-lint` image.  For example, the [GitHub Action](https://github.com/marketplace/actions/ansible-lint) uses the `quay.io/ansible/creator-ee:v0.6.0` image which ships with a much older version of `ansible-lint`.

Would it be possible to ship an official Docker image in order to be able easily use this?  I'm going to start building something in an outside repository, but happy to contribute that in if it's something the project wants.

**Comment by ssbarnea** [comment, marked as answer]
@mnaser We did not update the creator-ee image on purpose because the lastest two releases of the linter contained some regressions around `jinja[spacing]` but I am planning to make a bugfix release today, which should be good enough for inclusion in creator-ee.

Shortly, we do not include every release of the linter as we prefer to have it bit more stable. Still, thanks for raising the question.

**Reply by ssbarnea** [reply]
The linter is out but I discovered that ansible builder cannot install foreman collection. Basically this means that I will have to remove it from the container, for a while... My 10days PTO starts in half an hour so I cannot guarantee a fast fix.

**Reply by mnaser** [reply]
No worries, thanks for the context around that.  I appreciate it.
